src/csvFiles/Audit.csv

```diff
@@ -0,0 +1,128 @@
+ActionName,TimeStamp
+CustomerLogin,2022-04-30T13:23:47.700814
+CustomerMenu,2022-04-30T13:23:52.364147
+ViewProfile,2022-04-30T13:23:56.406295
+AddLoan,2022-04-30T13:24:04.757649
+ViewProfile,2022-04-30T13:24:12.760161
+CustomerLogin,2022-04-30T13:24:53.924604
+CustomerMenu,2022-04-30T13:25:06.565097
+AddLoan,2022-04-30T13:25:29.382228
+AdminLogin,2022-04-30T13:25:32.597960
+AdminMenu,2022-04-30T13:25:39.815237
+ViewCustomersThatHaveLoans,2022-04-30T13:25:44.188898
+ActionName,TimeStamp
```

Comment:
ActionName, TimeStamp ar trebui sa fie doar pe prima linie a csv-ului ca si "cap de tabel", in rest doar perechile de valori pentru ele mai jos

Reply:
Modific acum, dar am început deja sa scriu la etapa 3 și o sa se vadă în celelalte fișiere modificate. Este vreo problema?

Reply:
nu e nicio problema :)

Reply:
Pot sa dau merge dacă totul este ok?